pkg/build/version-handler.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"os"
+
+	"github.com/onosproject/onos-docs/pkg/common"
+
+	"github.com/onosproject/onos-docs/pkg/repository"
+	utils "github.com/onosproject/onos-docs/pkg/utils"
+	"gopkg.in/src-d/go-git.v4"
+)
+
+// TagName repo tag name
+type TagName int
+
+// latest tag name
+const (
+	Latest TagName = iota
+)
+
+func (t TagName) String() string {
```

Comment:
I think this version shoudl be read and specified elsewhere and not hardcoded

Reply:
Yes. I am aware of it. I am still thinking where is the best place to define the latest version without introducing redundant information.